addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -99,7 +209,7 @@
          t-att-data-selector="_selector"
          t-att-data-exclude="_exclude"
          t-att-data-target="_target">
-        <we-select string="Newsletter" data-attribute-name="listId"></we-select>
+        <we-select string="Newsletter" data-attribute-name="listId" data-dependencies="email"></we-select>
```

Comment:
That would not be a specific name enough and difficult to grep (in general we at least finish them by "_opt" I think, could be nice to use the same convention). Also I see in other commit you add a template and have to override the data-dependencies, maybe you could use `!form(_opt)` here instead of listing all the other ones and have to xpath to override that dependencies (not sure it's better but telling you so that you know it's a possibility).